x-pack/plugins/fleet/server/saved_objects/index.ts

```diff
@@ -742,7 +742,7 @@ export const getSavedObjectTypes = (
       name: DOWNLOAD_SOURCE_SAVED_OBJECT_TYPE,
       indexPattern: INGEST_SAVED_OBJECT_INDEX,
       hidden: false,
-      namespaceType: useSpaceAwareness ? 'single' : 'agnostic',
+      namespaceType: 'agnostic',
```

Comment:
@kpollich Should download source be namespace aware? fleet server hosts and outputs are not, it seems a little weird to me these ones are.

Reply:
No I think all of the settings should be global - this is probably just something I missed in the initial POC PR. Good call out 🙂